cmake/Buildosqp-eigen.cmake

```diff
@@ -0,0 +1,15 @@
+# Copyright (C) Fondazione Istituto Italiano di Tecnologia
+# CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+include(YCMEPHelper)
+include(FindOrBuildPackage)
+
+find_or_build_package(osqp QUIET)
+
+ycm_ep_helper(osqp-matlab TYPE GIT
+              STYLE GITHUB
+              REPOSITORY dic-iit/osqp-matlab.git
```

Comment:
This is a left-over I guess: repo should be `dic-iit/osqp-matlab-cmake-buildsystem`.

Reply:
Fixed.